apps/multi-tenancy-activate-deactivate/run.go

```diff
@@ -675,6 +614,29 @@ func test2() {
 	log.Println("TEST 2 finished. OK")
 }
 
+func updateTentantWithRetry(client *wvt.Client, className string, tenants Tenants) error {
```

Comment:
Is this one necessary ? We should only need retry on the function below where we check the object count but not this one where we write them, weaviate should be able to handle that.

I just don't want to introduce unecessary retries that could hide an issue.

Reply:
I added it because I saw that after adding the other retry(checking the object count) the test still failed in later stages because Weaviate wasn't able to handle new requests due to high CPU usage and it may reject the requests.

Reply:
@moogacs helped me to analyze the scenario :)